modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -610,12 +639,7 @@ export default class GPUGridAggregator {
       isInstanced: true,
       instanceCount: 0,
       attributes: {
-        position: new Buffer(gl, {
-          data: new Float32Array([0, 0]),
-          accessor: {
-            size: 2
-          }
-        })
+        position: this.state.resources['all-aggregation-buffer']
```

Comment:
You should be able to use `{constant: true, value: [0, 0]}`

Reply:
Yep, changed to [0, 0]